RMG/4Day/완주하지 못한 선수.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * 마라톤 경기에서 참가자와 완주자를 입력받고 완주하지 못한 사람을
+ * 찾는 함수입니다. (탈락자는 무조건 1명입니다.)
+ * solution(["leo", "kiki", "eden"], ["eden", "kiki"]); // "leo"
+ * solution(["marina", "josipa", "nikola", "vinko", "filipa"], ["josipa", "filipa", "marina", "nikola"]); // "vinko"
+ * solution(["mislav", "stanko", "mislav", "ana"], ["stanko", "ana", "mislav"]); // "mislav"
+ */
+
+function solution(participant, completion) {
+  // 빈 객체를 생성합니다.
+  let obj = new Object();
```

Comment:
```suggestion
  const obj = {};
```
객체를 생성하는 방법으로 생성자 함수를 사용 할 수 있지만, 일반적으로는 객체 리터럴({})을 통해 객체를 생성하는게 일반적입니다.

Reply:
빈 객채라 new Object()를 이용하여 새로 만드는 것이 코드 이해하기에 더욱 용이하다고 판단하여 작성하였습니다 !!